src/store/ChartStore.js

```diff
@@ -108,6 +109,10 @@ class ChartStore {
         this.chartContainerHeight = this.chartHeight - offsetHeight;
     }
 
+    notify(message) {
+        if (this.messaging) { this.messaging(message); }
```

Comment:
consider `onMessage` instead of `messaging`. `messaging` implies a status, not an event handler.